src/_P040_ID12.ino

```diff
@@ -121,7 +121,7 @@ boolean Plugin_040(uint8_t function, struct EventStruct *event, String& string)
               // temp woraround, ESP Easy framework does not currently prepare this...
               taskIndex_t index = INVALID_TASK_INDEX;
               for (taskIndex_t y = 0; y < TASKS_MAX; y++)
-                if (Settings.TaskDeviceNumber[y] == PLUGIN_ID_040)
+                if (Settings.TaskDeviceNumber[y].value == PLUGIN_ID_040)
```

Comment:
I'd expected that ` pluginID_t::toPluginID(PLUGIN_ID_040)` to be used here as well...? (Or should the other plugins use `.value`?)

Reply:
You can do both.
But since the static function toPluginID takes resources, it is more lightweight to do it the other way around.